tests/core/test_solve.py

```diff
@@ -742,9 +686,10 @@ def test_broken_install():
     #         'channel-1::dateutil-2.1-py27_1',
     #         'channel-1::flask-0.9-py27_0',
     #         'channel-1::scipy-0.12.0-np16py27_0',
-    #     ]
-    #     assert tuple(final_state_2) == tuple(solver._index[Dist(d)] for d in order)
-    #     assert r.environment_is_consistent(order)
+    #         'channel-1::pandas-0.11.0-np16py27_1',
+    #     )
+    #     assert convert_to_dist_str(final_state_2) == order
+    #     assert solver._r.environment_is_consistent(final_state_2)
```

Comment:
Looks like we've removed a lot of behavior testing/enforcement here too.  Just want to make sure that was intentional.

Reply:
Some of this can/should go back. I'll re-examine it before merging.  Thanks for pointing it out.